### DISCUSSION TITLE
Are blog entry contributions from community accepted?

### DISCUSSION THREAD

**Discussion by lohbe** [ROCm/rocm-blogs#12] [General]
If yes, are there any guidelines/howtos? Thanks in advance.

**Comment by jychang48** [comment]
The plan is to eventually accept blog entries from the community, but as this rocm blog initiative is still in its early stages we are still hashing out the overall workflow for external contributions. Guidelines for this will be posted once it's been finalized

**Reply by saadrahim** [reply]
Any ideas on what you want to contribute to the blogs repository? A future contribution guideline for community contributions will encourage that the subject matter of the blog is precleared with the ROCm blog team first to avoid disappointment.

AMD will retain editorial and publication discretion for all community contributions. There is no guarantee that your final blog will be accepted. 

Despite these statements, expect encouragement and support for any contributions.

**Reply by lohbe** [reply]
Thanks for the initial guidance. The content I'm proposing goes towards tracking ROCm features of interest to enterprises - e.g. manageability, RAS, monitoring/observability.
